components/application/PostAcceptanceForm.tsx

```diff
@@ -43,7 +43,8 @@ const AttendingForm: React.FC<AttendingFormProps> = ({ setAttendingState }) => {
   };
   return (
     <>
-      <p>
+      <p>This RSVP form is DUE for Round 1 Acceptances: Jan 21 11:59 PM EST.</p>
```

Comment:
I already added this , it should be at the topmost section of the post acceptance form in the questions.tsx